src/lib/utils.js

```diff
@@ -352,6 +352,36 @@ const ErrorCode = {
 
 const parseConfigFile = (config_path) => YAML.safeLoad(fs.readFileSync(config_path, 'utf8'));
 
+/**
+ * Check whether the path already exist.
+ * @param {String} path
+ * @return {Boolean}
+ */
+function folder_exists(path) {
+  try {
+    const stat = fs.statSync(path);
+    return stat.isDirectory();
+  } catch(_) {
+    return false;
+  }
+}
+
+/**
+ * Check whether the file already exist.
+ * @param {String} path
+ * @return {Boolean}
+ */
+function file_exists(path) {
```

Comment:
camelCase??

Reply:
🤕  trying to avoid conflicts ... but you are right.